assets/Interface/DistractionScreenGUI.xml

```diff
@@ -28,34 +34,48 @@
                 <control name="label" id="GLabel19" width="60px" x="11" y="333" text="Fog" height="25px" font="aurulent-sans-16.fnt"/>
                 <control name="label" id="GLabel17" width="60px" x="15" y="248" text="Snow" height="25px" font="aurulent-sans-16.fnt"/>
                 <control name="label" id="GLabel18" width="60px" x="15" y="158" text="Rain" height="25px" font="aurulent-sans-16.fnt"/>
+				
                 <control name="button" id="StartButton" childLayout="center" x="529" width="120px" y="560" label="Start" height="31px">
 					<interact onClick="clickStartButton()" />
 				</control>
+				<control name="button" id="InstructionButton" childLayout="center" x="529" width="120px" y="480" label="Instruction" height="31px">
+					<interact onClick="clickInstButton()" />
```

Comment:
Yes, first i create control for the button and then add interact function to it which is implemented in java. Default values are read from the code, will make it to be from txt.